js/request.js

```diff
@@ -150,7 +150,6 @@ exports.request = function(req, res, opts, handler) {
       var terminationMessage = 'Terminating server at http://[localhost]:' + opts.port + ' for ' + opts['module-path'] + ' / ' + opts.handler;
       res.end(terminationMessage + '\n');
       console.info(terminationMessage);
-      server.close();
```

Comment:
It used to not be a `NOOP`. My guess is that a refactor that moved the request handling to a separate file introduced that as a bug.